src/irr/asset/bawformat/TypedBlob.cpp

```diff
@@ -424,57 +434,4 @@ void TypedBlob<FinalBoneHierarchyBlobV0, CFinalBoneHierarchy>::releaseObj(const
 }
 
 
-
-template<>
```

Comment:
don't delete, move to legacy

Reply:
nah, those functions are not needed, just blob type